lib/recipe.rb

```diff
@@ -157,3 +195,5 @@
   )
   action [:enable, :start]
 end
```

Comment:
Fixed (I'd left it that way to keep my rebases from getting painful but that's not needed now)